modules/auxiliary/dos/smb/smb_loris.rb

```diff
@@ -22,23 +22,23 @@ class NbssHeader < BinData::Record
     grinds to a halt. This vulnerability was originally disclosed by Sean Dillon
     and Zach Harding.
 
-    DISCALIMER: This module opens a lot of simultaneous connections. Please check
+    DISCLAIMER: This module opens a lot of simultaneous connections. Please check
     your system's ULIMIT to make sure it can handle it. This module will also run
     continuously until stopped.
   },
   authors: [
-      'thelightcosine',
-      'Adam Cammack <adam_cammack[at]rapid7.com>'
+    'thelightcosine',
+    'Adam Cammack <adam_cammack[at]rapid7.com>'
   ],
   date: '2017-06-29',
   references: [
     { type: 'url', ref: 'https://web.archive.org/web/20170804072329/https://smbloris.com/' },
-    { type: 'aka', ref: 'SMBLoris'}
+    { type: 'aka', ref: 'SMBLoris' }
   ],
   type: 'dos',
   options: {
-    rhost: {type: 'address', description: 'The target address', required: true, default: nil},
-    rport: {type: 'port', description: 'SMB port on the target', required: true, default: 445},
+    rhost: { type: 'address', description: 'The target address', required: true, default: nil },
```

Comment:
Not a blocker; looks like this module has issues with inline options, i.e. `run rhost=x.x.x.x`